.codeclimate.yml

```diff
@@ -1,4 +1,7 @@
 version: "2"
+checks:
+  similar-code:
+    enabled: false
```

Comment:
in [cloudscale-metrics-collector](https://github.com/vshn/cloudscale-metrics-collector/pull/21#pullrequestreview-1183842033) @ccremer mentioned that I should remove all codeclimate stuff. Not sure if that's a general rule for vshn repos but something worth considering.

Reply:
Cloudscale collector was developed a bit differently than usual go code. I think that was the main reason why we had to remove it. We had some issue with code climate here as well and we had to adjust a bit the config as we thought that we should add more tests and further refactoring when we will merge both metrics collectors.

Reply:
I would remove codeclimate whenever you have some time for repo cleanup when you touch a Go repo at VSHN. In our bootstrap repo we also replaced codeclimate already.

> I think that was the main reason why we had to remove it.

No, we're replacing codeclimate because it seems to abandon the Go language support.

Reply:
> No, we're replacing codeclimate because it seems to abandon the Go language support.

Is there an official page for that?

Reply:
Does anyone know why `codeclimate` is still in the checks of the PR, even though I deleted everything related to it? Do we need to merge to master before it's completely gone or am I missing something?